docs/adr/0003-management_command_delete_signals.md

```diff
@@ -0,0 +1,134 @@
+# Django management command to delete Signals
+
+Date: 2022-05-27
+Author: David van Buiten
+
+## Status
+
+2022-05-27 - Draft (David van Buiten)
```

Comment:
(Not a blocker, as I see ADRs mostly as documentation.)

An ADR is a discussion piece, this is more documentation of the current implementation. I guess the status should not be draft when this is merged to the master branch.

Reply:
It is a draft until the PR is merged, so just before (or if I forget to do it just after) this will change to approved.